endpointmanager/cmd/main.go

```diff
@@ -43,28 +45,24 @@ func setupConfig() {
 	viper.SetDefault("dbpass", "postgrespassword")
 	viper.SetDefault("dbname", "lantern")
 	viper.SetDefault("dbsslmode", "disable")
-	viper.SetDefault("logfile", "endpointmanagerLog.json")
-}
-
-func initializeLogger() {
-	log.SetFormatter(&log.JSONFormatter{})
-	f, err := os.OpenFile(viper.GetString("logfile"), os.O_WRONLY|os.O_CREATE, 0755)
-	if err != nil {
-		log.Fatal("LogFile creation error: ", err.Error())
-	}
-	log.SetOutput(f)
 }
 
 func main() {
 	var err error
 
 	setupConfig()
-	initializeLogger()
 
 	store, err := postgresql.NewStore(viper.GetString("dbhost"), viper.GetInt("dbport"), viper.GetString("dbuser"), viper.GetString("dbpass"), viper.GetString("dbname"), viper.GetString("dbsslmode"))
 	if err != nil {
 		panic(err.Error())
 	}
 	defer store.Close()
 	fmt.Println("Successfully connected!")
+
+	ctx := context.Background()
+	client := &http.Client{}
```

Comment:
should add a timeout to the default client https://medium.com/@nate510/don-t-use-go-s-default-http-client-4804cb19f779